Bottom: 89da7110d11eacd6bc5744af902e2e24e8a49539
Top:    2b458f4817ad842a595a2e473a433410573500c0
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 10:47:49 +0800

Refresh of debug.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index adf302a..1efdc9e 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -1300,9 +1300,8 @@ int parse_mount(struct autofs_point *ap, const char *name,
 	 * doesn't confuse the options parse.
 	 */
 	if (strstr(p, "-hosts")) {
-		char *pos = p;
-		*pos++ = 'h'; *pos++ = 'o'; *pos++ = 's'; *pos++ = 't';
-		*pos++ = ':';
+		*p++ = 'h'; *p++ = 'o'; *p++ = 's'; *p++ = 't';
+		*p++ = ':';
 	}
 
 	/* Deal with 0 or more options */
